sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala

```diff
@@ -585,8 +585,8 @@ private[sql] object ParquetSchemaConverter {
     Types.buildMessage().named(ParquetSchemaConverter.SPARK_PARQUET_SCHEMA_NAME)
 
   def checkFieldName(name: String): Unit = {
-    // ,;{}()\n\t= and space are special characters in Parquet schema
-    if (name.matches(".*[ ,;{}()\n\t=].*")) {
+    // ,;{}\n\t= and space are special characters in Parquet schema
+    if (name.matches(".*[ ,;{}\n\t=].*")) {
```

Comment:
Oh, is this change required?

Reply:
Same question. this looks like a check for special chars in Parquet schema. Why need to change it?

Reply:
because in `ParquetScan` schema, I have something like this: `Max(col)`, `Min(col)`, so I need to remove the restriction for `()`

Reply:
Not sure if `()` will create some issues at Parquet side?